### DISCUSSION TITLE
I've never felt more productive

### DISCUSSION THREAD

**Discussion by heibel** [blitz-js/blitz#2737] [Kind Words]
So I gave Blitz a try after trying to implement something like RPC myself; and I must say Blitz is 🔥!
Whipped up a MVP in an afternoon. Everything has a very familiar feeling and a place in the project (graphql-based CMS integration, auth, etc..) So many thanks and hoping to contribute to this awesome project very soon.

Keep up the good work.

**Comment by flybayer** [comment]
Awesome, thanks for kind words! 😊
